Mobile_Detect.php

```diff
@@ -1450,13 +1450,17 @@ public function is($key, $userAgent = null, $httpHeaders = null)
      * the User-Agent string.
      *
      * @param $regex
-     * @param  string $userAgent
+     * @param string $userAgent
      * @return bool
      *
      * @todo: search in the HTTP headers too.
      */
     public function match($regex, $userAgent = null)
     {
+        if (!\is_string($userAgent) && !\is_string($this->userAgent)) {
```

Comment:
@julienbornstein thank you! 👍 Are the qualifiers really required since they are build in methods?

Reply:
sorry @serbanghita  I'm not sure to understand your question ? can you be more specific ?

Reply:
@julienbornstein I was refering to the usage of `\is_string` instead of `is_string`

Reply:
oh I see, it's to avoid collision if another is_string() function is declared. I'm not sure it's usefull herebut it's an habit that I have ... you can remove them if you want. 

Stackoverflow: [Should I use a backslah before global functions in PHP even if is not needed?](https://stackoverflow.com/questions/62568683/should-i-use-a-backslah-before-global-functions-in-php-even-if-is-not-needed)